backend/src/main/java/wooteco/retrospective/presentation/attendance/AttendanceController.java

```diff
@@ -24,8 +33,27 @@ public AttendanceController(AttendanceService attendanceService) {
     @PostMapping("/time")
     public ResponseEntity<AttendanceResponse> postTime(@Valid @RequestBody AttendanceRequest attendanceRequest) {
         AttendanceResponse attendanceResponse = AttendanceResponse.of(
-                attendanceService.postAttendance(attendanceRequest));
+            attendanceService.postAttendance(attendanceRequest));
 
         return ResponseEntity.ok().body(attendanceResponse);
     }
+
+    @DeleteMapping("/time")
+    public ResponseEntity<Void> deleteTime(@Valid @RequestBody AttendanceRequest attendanceRequest) {
+        attendanceService.deleteAttendance(attendanceRequest);
+
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping("/time/{time}")
```

Comment:
```suggestion
    @GetMapping("/time/{timeId}")
```
timeId를 정확하게 명시해 주는게 어떨까요?